phpcs.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
-<ruleset name="CakePHP Bake">
+<ruleset name="CsvView">
     <config name="installed_paths" value="../../cakephp/cakephp-codesniffer" />
-    <arg value="nps"/>
+    <arg value="s"/>
```

Comment:
Whats the advantage of np removed?

Reply:
Don't remember what `n` does but `p` is for "progress" and it was causing problems for the checkstyle to PR format conversion using `cs2pr`.

Reply:
-n    Do not print warnings (shortcut for --warning-severity=0)

I guess we dont need to exclude them if they dont bother us, so fair enough